src/SonarScanner.MSBuild.Common/CommandLine/CommandLineFlagPrefix.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SonarScanner.MSBuild.Common.CommandLine
+{
+    public static class CommandLineFlagPrefix
+    {
+        private static readonly char[] _prefixes = new[] { '-', '/' };
+
+        public static string[] GetPrefixFlag(string[] flags)
```

Comment:
Making the argument a `params` argument would make it cleaner to call with single/multiple arguments.

Reply:
I had a doubt whether i should put it or not, but finally not, so i brought it back.